.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ['*']
```

Comment:
```suggestion
```

Reply:
Only actions on the main branch will now trigger a Github CI run.